### DISCUSSION TITLE
Email com o dominio da APYB para PUG-PE

### DISCUSSION THREAD

**Discussion by naanadr** [apyb/comunidade#83] [Pergunta]
Olá pessoal, tudo belezinha?

Faço parte do PUG-PE e gostaria de saber se teria a possibilidade de ter um e-mail dentro do dominio da apyb (exemplo: `pugpe@python.org.br`)?

**Comment by rougeth** [comment]
Olá @naanadr! A gente não consegue criar e-mails para comunidades no @python.org.br. Tem algum motivo específico que vocês precisam de um e-mail para o pug-pe?

O que nós poderíamos avaliar, seria criar o sub domínio `@pugpe.python.org.br` e apontar para algum serviço de e-mail usado por vocês. Porém, vocês precisariam cuidar desse serviço (hospedagem, custos, manutenção), a APyB só conseguiria, neste momento, fornecer o domínio.

**Reply by naanadr** [reply]
Oiie @rougeth 😄 

> Tem algum motivo específico que vocês precisam de um e-mail para o pug-pe?

Ficamos sem e-mail esse ano, tínhamos um email que era no domínio `@pug.pe` , mas ele estava no nome de alguém antigo da comunidade e acho que deixaram de pagar pelo domínio ou o google começou a cobrar (não sei). Desde então ficamos sem e-mail oficial 😢 

> vocês precisariam cuidar desse serviço (hospedagem, custos, manutenção), a APyB só conseguiria, neste momento, fornecer o domínio.

A APyB nesse caso não consegue nos ajudar a ter o google non-profit? 🤔

**Reply by tiidadavena** [reply]
O Google for NonProfits precisa de estatuto ou de algum documento comprobatório de que trata-se de uma ONG, neste caso, a APyB poderia ajudar com uma carta de validação de que a PUG-PE é uma comunidade sem fins lucrativos.

**Reply by naanadr** [reply]
Seria ótimo. O que precisamos fazer aqui do lado do PUG-PE? Precisamos pagar pelo domínio? Desculpa se estou fazendo perguntas idiotas, eu realmente não sei como que funciona essas coisas.

**Reply by naanadr** [reply]
up @rougeth @tiidadavena

**Reply by tiidadavena** [reply]
Opa, podes perguntar à vontade! O processo também está sendo novo para a gente <3
Segundo a Google, os critérios de elegibilidade para instituições brasileiras são: https://support.google.com/nonprofits/answer/3215869?hl=pt-BR&ref_topic=3247288#zippy=%2Cbrasil.

Neste caso, para o PUG-PE não precisar de abrir CNPJ e nem redigir um estatuto, creio que vale a pena solicitar o [Google for Nonprofits](https://accounts.google.com/AccountChooser/signinchooser?continue=http%3A%2F%2Fwww.google.com%2Fnonprofits%2Faccount%2Fhome&flowName=GlifWebSignIn&flowEntry=AccountChooser) e anexar uma declaração de que a PUG é uma organização civil sem fins lucrativos e que é apoiada pela APyB.

Creio que ajudaria bastante anexar as redes sociais e a Diretoria pode ajudar também.

**Reply by naanadr** [reply]
Entendi 😄 a diretoria poderia então redigir essa declaração e assinar ela? Vou tentar enviar a solicitação e ver qual o retorno deles.

Além disso... abri o formulário do Nonprofits, e ele solicita um **Número de Registro**, não temos isso. Imagino que não podemos utilizar a da APyB, correto? 🤔 Vou tentar enviar o formulário com o número de registro igual a `-` , aparentemente ele aceita 😆 

![image](https://github.com/apyb/comunidade/assets/21251472/e83baa34-e100-457f-9f97-3500ddad37ae)

![image](https://github.com/apyb/comunidade/assets/21251472/51f1df08-b6d1-476d-a1e7-d6ba2cc028e8)
